From 0449b7e42809db16df87c3fbe95c37a756c04587 Mon Sep 17 00:00:00 2001
From: Wolfgang Dautermann <dauti@users.sourceforge.net>
Date: Tue, 22 Jul 2025 21:36:43 +0200
Subject: [PATCH] Fix compiling error with wxWidgets 3.3.0.

(See https://wxwidgets.org/help/msgid-literals/). Translating a
non-constant string (in a wxm header?) does not make much sense.

And this breaks compiliation with wxWidgets 3.3.0.
---
 src/main.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main.cpp b/src/main.cpp
index e39a0d035..0afed1975 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -524,13 +524,13 @@ void MyApp::NewWindow(const wxString &file, bool evalOnStartup,
     while (!istream.Eof()) {
       line = textIn.ReadLine();
       if ((line.StartsWith("/*")) || (line.EndsWith("*/"))) {
-        initialContents += _(block);
+        initialContents += block;
         initialContents += line + "\n";
         block.Clear();
       } else
         block += line + "\n";
     }
-    initialContents += _(block);
+    initialContents += block;
   }
   wxMaxima *frame = new wxMaxima(NULL, wxID_ANY, title, file, initialContents);
   frame->EvalOnStartup(evalOnStartup);
